app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardViewModel.kt

```diff
@@ -67,34 +66,15 @@ class DashboardViewModel(
     }
 
     fun observeContactsHealthStatus(): Observable<HealthStatusData> {
-        return Observables.combineLatest(
```

Comment:
Double check if any of the old logic might be still required.

Reply:
@falkorichter , @dusanjencik Do you know how this screen (zpl://screen?pid=5e6d1a927691ed15113b37a0&sid=5e7159f573c497b6352fd19f) should look like after the G/A update? Based on the information that we need to display in this screen we can know what data we should pass in the `HealthStatusData.ContactsSicknessInfo`